modules/react/combobox/lib/ComboboxMenuItem.tsx

```diff
@@ -57,17 +60,34 @@ export const useComboboxMenuItem = composeHooks(
   useListItemRegister,
   useListItemAllowChildStrings
 );
+const comboboxMenuItemIconStencil = createStencil({
+  base: {
+    alignSelf: 'start',
+  },
+});
+
+export const ComboboxMenuItemIcon = createSubcomponent('span')({
+  modelHook: useComboboxModel,
+})<ExtractProps<typeof SystemIcon>>(({...elemProps}, Element) => {
```

Comment:
```suggestion
})<ExtractProps<typeof SystemIcon>>((elemProps, Element) => {
```
no destructuring needed